src/lib.rs

```diff
@@ -113,7 +115,7 @@ impl AdvisoryFileLock {
 impl Drop for AdvisoryFileLock {
     fn drop(&mut self) {
         if let Err(err) = self.unlock() {
-            log::error!("Unlock_file failed during dropping: {}", err);
+            panic!("unlock_file failed during dropping: {}", err);
```

Comment:
I prefer not to panic here, would you mind changing this back to `log::error`?

Reply:
Hmm, yeah, I don't know why I changed that